app/api/ringle_music/v1/user_api.rb

```diff
@@ -0,0 +1,151 @@
+module RingleMusic
+    module V1
+        class UserApi < Grape::API
+            version 'v1', using: :path
+            format :json
+            prefix :api
+
+            resource :users do
+
+                get :getinfo do
```

Comment:
GET users/info가 나을 것 같습니다.
'REST API design guide' 요런 키워드로 서치해보시면 좋을 것 같습니다 :)

https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/
Use Nouns Instead of Verbs in Endpoints / Name Collections with Plural Nouns 요런 컨벤션들 참고하시면 좋습니다.